src/components/Notifications/BehaviorGroup/BehaviorGroupCard.tsx

```diff
@@ -61,11 +63,12 @@ const BehaviorGroupCardLayout: React.FunctionComponent<BehaviorGroupCardLayout>
                 <CardActions>
                     <Dropdown
                         onSelect={ switchOpen }
-                        toggle={ <KebabToggle onToggle={ setOpen } isDisabled={ !props.dropdownItems } /> }
+                        toggle={ <KebabToggle onToggle={ setOpen } isDisabled={ !props.dropdownItems || !rbac.canWriteNotifications } /> }
```

Comment:
This, and the previous lines are part of other PR. Could you remove them off this PR ?